include/find_deletions/deletion_finding_and_printing.hpp

```diff
@@ -1,16 +1,21 @@
 #pragma once
 
 #include <seqan3/std/filesystem>
+#include <seqan3/io/stream/concept.hpp>
 #include <fstream>
+#include <iostream>
 #include <vector>
 
 #include "junction.hpp"
 
 /*! \brief Detects deletions out of the junction file.
  *
  * \param junction_file_path input junction file
- * \param output_file_path output vcf file
+ * \param out_stream output stream
  *
  * \details Extracts deletions out of given breakends / junctions.
  */
+void find_and_print_deletions(std::filesystem::path const & junction_file_path, std::ostream & out_stream);
+
+//!\overload
 void find_and_print_deletions(std::filesystem::path const & junction_file_path, std::filesystem::path const & output_file_path);
```

Comment:
Can you insert a line break? We want to have a limit of 120 characters per line.